src/lib/transition_handler/block_sink.ml

```diff
@@ -64,8 +64,10 @@ let push sink (`Transition e, `Time_received tm, `Valid_cb cb) =
             in
             Mina_metrics.Block_latency.(
               Validation_acceptance_time.update processing_time_span)
-        | _ ->
-            () ) ;
+        | None ->
+            [%log error] "Validation timed out on block"
+              ~metadata:[ ("external_transition", Mina_block.to_yojson state) ]
```

Comment:
Call this "block" instead of "external_transition" (we are sunsetting that term).

Reply:
Same here: can you reference `$block` in the log message?